### ISSUE TITLE
Support for Catppuccin themes

### ISSUE THREAD

**Issue by codedemonuk** [usebruno/bruno#2189]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

I would like to add 4 themes to Bruno using the [open source Catpuccin colour scheme](https://github.com/catppuccin/catppuccin) in additional to the current light and dark themes that are currently in Bruno.

Is this something that would be welcomed or would any themes beyond light and dark be something reserved for the paid version of Bruno

### Mockups or Images of the feature

.

**Comment by MathieuDR** [comment]
Is this still on the table?

**Comment by anusree-bruno** [comment]
Yes, it is on our radar. We definitely want to support themes in Bruno, but we've been bogged down with other issues and feature requests, so we haven’t had a chance to implement it yet. That said, we're aiming to work on this in Q3 (July–September).

**Comment by Ostralyan** [comment]
Following up on this :)

**Comment by naman-bruno** [comment]
hey @Ostralyan We’ve rolled out theming in v3; you should check it out ([download page](https://www.usebruno.com/downloads))

<img width="1552" height="912" alt="Image" src="https://github.com/user-attachments/assets/31c87a83-3541-4b63-8336-8fadf9badd98" />
